### PR TITLE
Add some missing recipes

### PR DESCRIPTION
**Pull request by lynxx131**
Added in some missing crafting recipes:

- Bronze dust
- Vacum freezer
- Electric Forge hammer
- Implosion Compressor
- Multi smelter
- Heat Exchanger
- Distillation Tower
- Cracking Unit
- Battery buffer nine
- Fluid input and output hatches
- Item output hatch
- Energy hatch

This is an attempt address #135, but there's probably more recipes that I missed.

### CODE REVIEW

File: common/src/main/java/muramasa/gregtech/loader/crafting/MaterialCrafting.java
Comment (Trinsdar) [inline common/src/main/java/muramasa/gregtech/loader/crafting/MaterialCrafting.java]: This should be a shapeless recipe.

**Review by Trinsdar — CHANGES_REQUESTED** [review]

File: common/src/main/java/muramasa/gregtech/loader/crafting/MaterialCrafting.java
Comment (lynxx131) [reply common/src/main/java/muramasa/gregtech/loader/crafting/MaterialCrafting.java]: Fixed

**Review by Trinsdar — APPROVED** [review]

**Comment by lynxx131** [comment]
Is there anything this PR is waiting on before being merged?

**Pull request merged**
